code/modules/paperwork/records/info/waw.dm

```diff
@@ -373,3 +401,16 @@
 		"Upon observing the Abnormality, it has been confirmed that the abnormality attached to employees for a certain amount of time, and inflicted WHITE damage over that period. The abnormality is highly contagious and difficult to recontain.",
 		"The infection was potentially cured if the infected received RED damage at a very small chance, with the provided flamethrowers being a far more effective deterrent.",
 		"When the SP of an infested employee had depleted, the employee in question would panic, spreading the infection rapidly.")
+
+//Apex Predator
+/obj/item/paper/fluff/info/waw/predator
+	abno_type = /mob/living/simple_animal/hostile/abnormality/apex_predator
+	name = "D-04-146"
+	abno_info = list(
+		"When the work result was Normal, the Qliphoth Counter lowered with a normal probability.",
+		"When the work result was Bad, the Qliphoth Counter lowered.",
+		"When the employee working with Apex predator started work with less than max health, the work damage was drastically increased.",
+		"When the employee working with Apex predator died during work, the Qliphoth Counter lowered.",
```

Comment:
Currently checks for downed, which can include non-dead, critical agents. I do think this is good writing for it but it's not necessarily accurate. How you wish to handle this is up to you, including not changing it.